spec/std/system/group_spec.cr

```diff
@@ -0,0 +1,84 @@
+require "spec"
+require "system/group"
+require "system/user"
+
+{% if flag?(:darwin) || flag?(:openbsd) || flag?(:freebsd) %}
+  private GROUP_NAME = "wheel"
+  private GROUP_ID = 0
+  private GROUP_MEMBERS = ["root"]
+{% elsif flag?(:linux) %}
+  private GROUP_NAME = "root"
+  private GROUP_ID = 0
+  private GROUP_MEMBERS = [] of String
```

Comment:
My distro has root as a user of the rout group. These specs are pretty flakey, depending largely on how the system is set up instead of anything else :(

Reply:
What distro? I have tried to use the most standard.

Reply:
Ruby has an interesting approach. I will see if I can do this.
https://github.com/rubysl/rubysl-etc/blob/2.0/spec/getpwnam_spec.rb

Reply:
Arch linux.

Reply:
Retrieving values from system commands at runtime is probably the only way to ensure they are actually valid on the target system.